### PR TITLE
Fiche soutenance done + Header redesigned

### LINKED ISSUE #18: Remplir la fiche d’évaluation de la soutenance

### PR DESCRIPTION
**Pull request by anto-lcd**
Things added : 
- TA can now submit a grade of a presentation (Code has been refactored by @RomainMarques inside his branch feature/report to also add report grading)
- Header of TA redesigned
- Header of ETU copied from TA's
- Added FontAwesome on TA and ETU fronts
- Fixed various HTML errors and warns
- TA Disconnect func now working

### CODE REVIEW

File: efrei-internship-back/services/ei-service-internship-info/models/index.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
```
Comment (RomainMarques) [inline efrei-internship-back/services/ei-service-internship-info/models/index.js:5]: tu peux importer Sequelize du fichier où je l'ai déclaré (j'ai plus son nom en tête)

File: efrei-internship-back/services/ei-service-internship-info/models/internship.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+'use strict';
+const { Model } = require('sequelize');
```
Comment (RomainMarques) [inline efrei-internship-back/services/ei-service-internship-info/models/internship.js:2]: pareil ici

File: efrei-internship-back/services/ei-service-internship-info/routes/apiEvaluation.js [hunk 3]
```diff
@@ -2,7 +2,9 @@ const {router} = require('../initializer/initRouter.js');
 const {sequelize, QueryTypes} = require('../initializer/initSequelize.js');
```
Comment (RomainMarques) [inline efrei-internship-back/services/ei-service-internship-info/routes/apiEvaluation.js]: tu peux revert les changements ici

**Review by Jathoosh — APPROVED** [review]
Nice

**Pull request merged**